features/certificate-management-ingress.md

```diff
@@ -0,0 +1,222 @@
+[//]: # (The syntax preceeding this line is a comment marker used to help guide the author in populating this document)
+[//]: # (to github. Unlike HTML comments commonly used throughout istio.io documentation, this comment will not be rendered)
+[//]: # (by github. Comments must be separated by carriage return preceding and concluding the text and be a single line.)
+
+[//]: # (This is a living document representing the maturity of a feature. Completion of this template enables Istio work groups)
+[//]: # (to collect information on potential new functionality. This template should be completed before users are exposed to)
+[//]: # (any new experimental feature. Please complete this template during development.)
+
+[//]: # (The feature implementation section must be completed before submission of the document.)
+
+# Feature:
+
+[//]: # (All information in this section is mandatory.)
+
+**Feature name:**
+
+Certificate Management for Istio Ingress Gateway
+
+[//]: # (The name of the feature, e.g. Multiple control planes)
+
+**Primary lead(s):**
+
+Limin Wang
+
+[//]: # (The primary lead or leads responsible for the feature. These individuals serve as a point of contact for the feature.)
```

Comment:
Add Oliver Liu and Jimmy Chen who designed and implemented the feature. 

@myidpt @JimmyCYJ Please comment on the rest of the sections.

Reply:
Done.